src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -1168,15 +1169,55 @@ impl CatalogState {
                 ServiceProcessMetrics {
                     cpu_nano_cores,
                     memory_bytes,
+                    disk_usage_bytes,
+                    disk_capacity_bytes,
                 },
             )| {
+                // We only populate a `disk_usage` if we have all 5 of of:
+                // - a disk limit (so it must be an actual managed cluster with a real limit)
+                // - a reported disk capacity
+                // - a reported disk usage
+                // - disk capacity is less than disk_usage.
+                // - There are no overflows
+                //
+                //
+                // The `disk_usage` is augmented with the difference between
+                // the limit and the `capacity`, so users don't erroneously think they
+                // have more real usable capacity than they actually do.
```

Comment:
Can you explain this more? What's the difference between limit and capacity?

Reply:
its something like 1-3%, depending on the size. Its the LVM and fs overhead

Reply:
Can you explain that in the comment?

Reply:
yes!